### ISSUE TITLE
Feature Requested: add remain quantity to mainOrders

### ISSUE THREAD

**Issue by YanB25** [wwyf/Waterfall-backend#14]
Add `remain_quantity` to all the mainOrders, 
remain_quantity = quantity - current_supply, for example
```
{
  "code": 0,
  "data": {
    "orders": [
      {
        "current_supply": 11,
        "quantity": 50,
        "remain_quantity": 39,
      }
    ]
  }
}
```

**Comment by wwyf** [comment]
OK

**Comment by wwyf** [comment]
fix by #25
